ci-operator/step-registry/openshift/windows/setup-wmco-konflux/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
```

Comment:
have mixed feelings about the dir structure proposed in this PR, I recommend

`ci-operator/step-registry/openshift/windows-qe/ ... ` 

so that the  `windows-qe` dir  contains all the QE related items, including OWNERS, etc

cc @sebsoto